src/main/java/com/softteam/vocabuilder/util/validations/Validations.java

```diff
@@ -1,17 +1,21 @@
 package com.softteam.vocabuilder.util.validations;
 
+import com.softteam.vocabuilder.exections.VocabuilderIllegalArgumentException;
 import lombok.Data;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
 
 import java.util.UUID;
 
 @Data
+@Component
 public class Validations {
     public UUID validateUUIDType(String parameter) {
```

Comment:
I approved the PR with this validation but I didn't see well what it was, I think you don't need to validate the param you receive is valid, you would need to catch in case an Exception is thrown by the repository

Reply:
I tried to do something as a good practice of using a validation so as not to repeat code in each of the methods. I don't know to what extent it could be right or it is done in a project.